observation_portal/observations/models.py

```diff
@@ -110,7 +109,9 @@ class Observation(models.Model):
     def cancel(observations):
         now = timezone.now()
 
-        _, deleted_observations = observations.filter(start__gte=now + timedelta(hours=72)).delete()
+        observation_ids_to_delete = [observation.id for observation in observations if
+                                     observation.start >= now + timedelta(hours=72)]
+        _, deleted_observations = Observation.objects.filter(id__in=observation_ids_to_delete).delete()
```

Comment:
I can't remember exactly why I made these changes but I think it was to fix an issue after updating libraries.